src/_utils/item/style.ts

```diff
@@ -1,19 +1,40 @@
 import css from 'styled-jsx/css'
 
-import { space } from '_utils/branding'
+import { color, space } from '_utils/branding'
 
 export default css`
   :global(.kirk-item) {
     position: relative;
     display: flex;
     padding: 0;
-    padding-top: ${space.l};
-    padding-bottom: ${space.l};
+    padding: ${space.l} ${space.xl};
     align-items: center;
     flex: 1;
     border: 0;
     background: none;
   }
+  :global(.kirk-item.kirk-item--clickable:hover) {
+    background: ${color.pressed};
```

Comment:
why 'pressed' and not 'hover' (since it's not pressed - also if it was pressed, the status would be "active")